packages/core/utils/test/urlJoin.test.js

```diff
@@ -0,0 +1,21 @@
+// @flow strict-local
+
+import assert from 'assert';
+import urlJoin from '../src/urlJoin';
+
+describe('urlJoin', () => {
+  it('Should join two paths', () => {
+    let joinedUrl = urlJoin('/', './image.jpeg?test=test');
+    assert.equal(joinedUrl, '/image.jpeg?test=test');
+  });
+
+  it('Should join two paths with longer publicUrl', () => {
+    let joinedUrl = urlJoin('/static', './image.jpeg?test=test');
```

Comment:
Think it'd be worth adding another test like this one but for just `'image.jpeg?test=test'`?

Reply:
sounds like a great idea will add one